docs/index.rst

```diff
@@ -42,7 +42,7 @@ Visiting ``http://<hostname>:10000/lab?token=<token>`` in a browser loads Jupyte
 where ``hostname`` is the name of the computer running docker and ``token`` is the secret token printed in the console.
 Docker destroys the container after notebook server exit, but any files written to ``~/work`` in the container remain intact on the host.::
 
-    docker run --rm -p 10000:8888 -e JUPYTER_ENABLE_LAB=yes -v "${PWD}":/home/jovyan/work jupyter/datascience-notebook:33add21fab64
```

Comment:
Unfortunately, these docs are duplicated, so you need to change the docs here as well 🙁

Reply:
😭 I will do it.

Reply:
Done!